staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -236,7 +236,7 @@ func (s *specAggregator) RemoveAPIService(apiServiceName string) error {
 	defer s.mutex.Unlock()
 
 	if _, exists := s.specsByAPIServiceName[apiServiceName]; !exists {
-		return ErrAPIServiceNotFound
+		return nil
```

Comment:
This function cannot return an error anymore right? Would it make sense to to change the function signature so it's more clear?

Reply:
Sure, changed the signature.